docs/explanation/core/proposer-based-timestamps.md

```diff
@@ -55,44 +33,182 @@ The following protocols and application features require a reliable source of ti
 * IBC packets can use either a [timestamp or a height to timeout packet
  delivery](https://ibc.cosmos.network/v8/ibc/light-clients/updates-and-misbehaviour?_highlight=time#checkformisbehaviour).
 
-### Proposer Selects a Block Timestamp
+### Enabling PBTS
+
+The PBTS algorithm is **not enabled by default** in CometBFT v1.0.
+
+If a network upgrades to CometBFT v1.0, it will still use the BFT Time
+algorithm until PBTS is enabled.
+The same applies to new networks that do not change the default values for
+consensus parameters in the genesis file.
+
+Enabling PBTS requires configuring the [consensus parameters](#consensus-parameters)
```

Comment:
Can you provide the exact field name here? (i.e., "PbtsEnableHeight")

Reply:
You have to set all of them, haven't you?

Reply:
OK, I now see what you mean here.
Can you then just mention the three params at the end of this sentence, to make it more self-contained?

Reply:
Added.